config/default/views.view.search.yml

```diff
@@ -1,6 +1,6 @@
 uuid: a9ce491f-2b24-406d-8a94-025fe3bdd801
 langcode: en
-status: true
+status: false
```

Comment:
Did you try disabling the core `search` module as a whole? Just curious, we probably don't have to do it.

Reply:
It looks like the search module had never been installed.

Reply:
Oh interesting, you're right. This works then!